### ISSUE TITLE
Logo is not read out when hovered on a11y devices

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#865]
**Describe the bug**
When using ZoomText's Reader 'Mouse Echo Hover' setting: hovering precisely over the 'BBC News SVG', the text 'BBC News' is not read out. When you hover just above or below the SVG, it does read out 'BBC News'.

This is replicated when CSS is turned off and also when the focusable & aria-hidden attributes are removed from the SVG. This was also observed when using the 'SpeakIt' tool and the 'AprRdr' (occasionally). This is worth a little bit of investigation to see if we can fix this.

When testing ORB the BBC logo is read out, the BBC blocks are a background image, this issue might be fixed if we used a background image rather than an SVG or there might be another solution.

John Hardey (Assistive Technology Specialist) has logged this issue with ZoomText - Awaiting response from ZoomText - I suggest we decided on an approach for this before addressing the High Contrast bug as it may have a knock on effect

**To Reproduce**
Hover over the `BBC News` SVG on...
- ZoomText's 'Mouse Echo Hover' feature
- SpeakIt
- AprRdr

**Expected behaviour**
BBC News should be read out when hovered across all of the logo.

**Comment by BogdanDogaru** [comment]
@greenc05 Have you heard back from John Hardey regarding this bug?

**Comment by greenc05** [comment]
I chased John last week @BogdanDogaru and he hadn't heard and was going to chase them up, I will ask if he has a response now... (he's on leave today)

**Comment by greenc05** [comment]
I've filed this bug with ZoomText and am awaiting a response from them...

**Comment by greenc05** [comment]
I've now sent log files for this bug as requested by ZoomText - am awaiting a response from them...

**Comment by sareh** [comment]
@greenc05 Is there any update on the ZoomText bug?

**Comment by greenc05** [comment]
No @sareh will chase them again... Thanks

**Comment by greenc05** [comment]
Response from Vispero (who make ZoomText) "I have not heard anything as of yet from the engineers.  I have pinged them again for an update.  I believe it is still being looked into.  I do not have a time frame on a possible fix.  It is possible it may not be fixed until the next version of ZoomText.  When I hear something I will let you know.  Thanks."

**Comment by greenc05** [comment]
Just re-tested on the Pidgin frontpage with ZoomText (v10.11.7.4) and IE11 (v11.0.9600.19540) on Windows 7, and all is working as expected with the brand logo
